tests/files/features/Dockerfile

```
@@ -4,7 +4,7 @@ FROM ${UPSTREAM_REPO:-testlagoon}/node-10-builder:${UPSTREAM_TAG:-latest} as bui
 COPY package.json yarn.lock /app/
 RUN yarn install
 
-FROM ${UPSTREAM_REPO:-testlagoon}/node-10:${UPSTREAM_TAG:-latest}
+FROM ${UPSTREAM_REPO:-testlagoon}/node-10:${UPSTREAM_TAG:-non-root-cron}
```

```suggestion
FROM ${UPSTREAM_REPO:-testlagoon}/node-10:${UPSTREAM_TAG:-latest}
```
The upstream pr has now been merged and a release is on it's way

---

thanks, will update